src/Application.vala

```diff
@@ -115,8 +115,6 @@ public class Music.Application : Gtk.Application {
 
         settings.bind ("window-maximized", main_window, "maximized", SettingsBindFlags.SET);
 
-        // This needs to be done after window is constructed
-        // Else music plays but the queue seems empty
```

Comment:
I think this was a good comment actually. It's good to provide the reason we don't do this in `playback_manager`'s construct block